src/webview_provider/altimateWebviewProvider.ts

```diff
@@ -104,7 +104,7 @@ export class AltimateWebviewProvider implements WebviewViewProvider {
     syncRequestId,
     ...rest
   }: SendMessageProps) {
-    this._webview?.postMessage({
+    this._panel?.webview?.postMessage({
```

Comment:
i am assuming that `this._webview` is not getting used anymore

Reply:
it is used in some other places